### DISCUSSION TITLE
Calendar: Consistent hour format

### DISCUSSION THREAD

**Discussion by chrpeters** [vuetifyjs/vuetify#11916] [General]
Hello, 
I would like to use a consistent hour format in the calendar component. By default, full hours are displayed as "HH", whereas timestamps with hour and minute are displayed as "HH:mm". My goal is to always use the "HH:mm" format, so that "12" would display as "12:00". Anybody have a hint how to achieve this?

![large-Screenshot 2020-07-15 at 11 09 50](https://user-images.githubusercontent.com/7147576/88264624-5c2bf200-cccc-11ea-9c93-73794ecec977.png)

**Comment by basvdijk** [comment]
I ran into the same issue, hope someone has a solution

**Comment by CristianRoibu** [comment]
I would appreciate it too. A consistent 24h (HH:mm) format would be very nice.
My temporary workaround is using locale="de", but full hours are displayed as "HH Uhr".
Adding :short-intervals="false" solves the problem in days view, but not in events.
![codepen io_cristian-roibu_pen_GRqeJyM_editable=true editors=101%3Dhttps%3A%2F%2Fvuetifyjs com%2F](https://user-images.githubusercontent.com/31011883/99199790-e3343700-27a1-11eb-9211-a2605a80e2bf.png)


![codepen io_pen__ editable=true editors=101=https%3A%2F%2Fvuetifyjs com%2F](https://user-images.githubusercontent.com/31011883/99198906-d876a380-279b-11eb-973e-874caf285d5c.png)

https://codepen.io/cristian-roibu/pen/GRqeJyM?editable=true&editors=101%3Dhttps%3A%2F%2Fvuetifyjs.com%2F

**Comment by basvdijk** [comment]
Currently as a solution I replace the minutes with 01 when they are 00
